{snapshots} Run on iOS 11.0 instead of 11.2

romoore opened the pull request:
Since kokoro's Xcode 9.0 and 9.1 environments don't support iOS 11.2 simulators, our continuous build jobs are failing due to the new snapshot tests. Instead of using iOS 11.2, we can use iOS 11.0. It's the same iOS version we're using for unit tests and so wouldn't require any additional simulator runtimes for developers.

iOS 11.1 was considered but due to rendering differences compared to iOS 11.2, iOS 11.0 is preferred.

As a minor point, the snapshot tests were timing-out at 5 minutes during some builds on kokoro.

Fixes #6327

Commit:
{snapshots} Use iOS 11.1 instead of 11.2

Commit:
Switching to 11.0

Commit:
Switching to 11.0.1

Commit:
Switch version

Commit:
Switch to 11.1 - not diff from 11.2

material-automation commented:
bazel detected changes to the following targets:

```
//...
```

<!-- identifier: affected-targets -->

Commit:
Move images to 11.1 and change test implementation

Commit:
Revert "Move images to 11.1 and change test implementation"

This reverts commit 3af6a2a6a047f2c3da8ccc69e7bc66e4748efa54.

Commit:
Switch to 11.0

11.1 had rendering difference with 1 Arabic test character. Since
there's no reason to require 2 different iOS 11 runtimes for tests, we
can just stick with 11.0 which is already used for unit tests.
